genai_bench/user/openai_user.py

```diff
@@ -261,6 +261,29 @@ def parse_chat_response(
                 num_prefill_tokens, num_prompt_tokens, tokens_received = (
                     self._get_usage_info(data, num_prefill_tokens)
                 )
+                if not time_at_first_token:
```

Comment:
does the situation in the test `The first chunk has > 1 tokens:` usually happen with MTP scenario?